integration/test/roster_test.js

```diff
@@ -0,0 +1,310 @@
+const { describe, before, after, it } = require('mocha');
+const { v4: uuidv4 } = require('uuid');
+const RosterTestPage = require('../pages/RosterTestPage');
+const DriverFactory = require('../utils/DriverFactory');
+const Window = require('../utils/Window');
+
+describe('Roster Test', async function () {
+  const driverFactory = new DriverFactory();
+  let page;
+  let localWindow;
+  let remoteWindow;
+  let failureCount = 0;
+
+  const meetingId = uuidv4();
+  const localAttendeeName = 'local-attendee';
+  const remoteAttendeeName = 'remote-attendee';
+
+  before(async function () {
+    await driverFactory.build('Roster Test', 'sauce-chrome');
```

Comment:
So we only test with Chrome right now right? How do i run this locally with local grid?
Can we make this a parameter?

Reply:
It is a parameter now:
```js
async build(testName, host = this.config.host) {
  ...
}
```
So if you want to run the test locally, just update it to:

```js
    await driverFactory.build('Roster Test', 'chrome');
````

It would be great to have a `enum` here, but it's `.js` file, so can't do it

Reply:
>So we only test with Chrome right now right? 

We can test with Firefox as well:
```js
await driverFactory.build('Roster Test', 'firefox'); // Locally
await driverFactory.build('Roster Test', 'sauces-firefox'); // On Sauce Labs
```

Reply:
>If so, you may need to update the screenshot in the description.

Updated.

Reply:
> How do i run this locally with local grid?

Simply remove `sauce-`
For example, to test on Sauce Labs
```
await driverFactory.build('Roster Test', 'sauce-firefox'); // On Sauce Labs with Firefox
// Or
await driverFactory.build('Roster Test', 'sauce-chrome'); // On Sauce Labs with Firefox
```

To test locally:

````
await driverFactory.build('Roster Test', 'firefox'); // Locally with Firefox
// Or
await driverFactory.build('Roster Test', 'chrome'); // Locally with Firefox
```

Reply:
Thats what I meant. It is not good though. We need to have a way to do this in common line and don't need to change code.

Reply:
> Thats what I meant. It is not good though. We need to have a way to do this in common line and don't need to change code.

Got it. Could you explain this a little bit more? How should it works like finally? Like how should it works, and what is the use cases.

e.g.
Should we run command once and then test is running against all platforms and browsers. Or we could run the command one by one to do that?

Reply:
Can we merge this first and improve it in another PR, since it's an improvement.

Reply:
I disagree. It is a basic requirement and we should just do it :)

Reply:
Updated. 
In `/integration` folder, simply call `npm run test:roster` to trigger roster integration test.
By default the `host` is local chrome browser, the `platform` is `Windows 10`. The `browser version` is `latest`.

To specify the `host`, `platform` and `browser version`. Just call the commend line with named argument.
```sh
npm run test:roster -- --host='sauce-chrome' --browser-version='latest' --platform-name='Mac'
```

Reply:
Nice work